storage/sqlStorages/postgres/datasource.go

```diff
@@ -0,0 +1,87 @@
+package postgres
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/cube2222/octosql"
+	"github.com/cube2222/octosql/config"
+	"github.com/cube2222/octosql/physical"
+	"github.com/cube2222/octosql/storage/sqlStorages"
+	_ "github.com/lib/pq"
+	"github.com/pkg/errors"
+)
+
+type PostgresTemplate struct{}
+
+func (t PostgresTemplate) GetAvailableFilters() map[physical.FieldType]map[physical.Relation]struct{} {
```

Comment:
(I'm not sure) - Shouldn't this be ` t *PostgresTemplate` ?
Tbf never wrote a method without a pointer receiver

Reply:
It probably doesn't really matter so I'm just curious what's the purpose of not using pointer receiver